src/radical/entk/execman/base/resource_manager.py

```diff
@@ -139,6 +139,20 @@ def shared_data(self, data_list):
         self._shared_data = data_list
```

Comment:
Do we have a `setter` for `shared_data`? If so, should we update the docstring of `shared_data` adding a setter?